### ISSUE TITLE
missing regions from add datasource list

### ISSUE THREAD

**Issue by nassbar** [oracle/oci-grafana-metrics#53]
An update to the list of default regions with a more current/complete list is required for those of us using this plugin and aren't in the current regions listed.

Currently I am "hacking" the constants to give me the complete list.

*Before*
![image](https://user-images.githubusercontent.com/8792065/83944348-11e8d180-a842-11ea-8419-85cd4798e806.png)

*After*
![image](https://user-images.githubusercontent.com/8792065/83944354-17461c00-a842-11ea-8137-518f589eab76.png)

*Current Region List*
```
$ oci iam region list | jq -r '.data[] | .name'
ap-chuncheon-1
ap-hyderabad-1
ap-melbourne-1
ap-mumbai-1
ap-osaka-1
ap-seoul-1
ap-sydney-1
ap-tokyo-1
ca-montreal-1
ca-toronto-1
eu-amsterdam-1
eu-frankfurt-1
eu-zurich-1
me-jeddah-1
sa-saopaulo-1
uk-london-1
us-ashburn-1
us-phoenix-1
```

**Comment by nicebits** [comment]
I'm having the same problem, us-sanjose-1 is missing.

**Comment by huskerdhf** [comment]
Hi @nassbar,

This issue was addressed in the 3.0.5 version of the metrics plugin.

Please try the updated oci-grafana-metrics plugin to determine if your issue was addressed to your satisfaction. Let us know if you believe the issue is still not addressed. If no response is received within 30 days, this issue will be closed.

Thanks
